ITAQA/crawler/emilia_romagna.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Emilia-Romagna data downloader and parser
+"""
+
+from datetime import datetime, timedelta
+from functools import reduce
+import urllib
+import pandas as pd
+
+from ITAQA.core.Pollutants import Pollutant
+
+
+def get_csv(pollutant=Pollutant.UNSET, days=10):
+    """
+    Return a csv containing the latest N days of measurament of Emilia-Romagna
+    air quality (pollutant of choice)
+
+    Data origin: ARPA Emilia-Romagna
+    Website: https://apps.arpae.it/qualita-aria/bollettino-qa
+    """
+
+    # TODO: support arbitrary date ranges
+    # TODO: catch HTTPError for unavailable pages and drop them
+    # TODO: discard or warn if page contains "Dati in attesa di validazione"
+    end_date = datetime.today() - timedelta(2)
+
+    date_list = pd.date_range(end=end_date, periods=days).tolist()
+    date_string_list = [x.strftime('%Y%m%d') for x in date_list]
+
+    pd_singleday_list = []    # List of prepped tables for merging + output
+
+    base_url = 'https://apps.arpae.it/qualita-aria/bollettino-qa/'
+    for date_string in date_string_list:
+        url = base_url + date_string
+        html_doc = urllib.request.urlopen(url).read()
+
+        pd_table_list = pd.read_html(html_doc)
+        pd_table_list.pop(-1)    # Last table is just pollutants threshold values
+
+        for pd_table in pd_table_list:
+            # TODO: Improve - e.g. do most stuff as inplace
+            # TODO: Add robust-ish checks rather than "blind" hardcoded indices?
+            # Drop last columns (threshold overshooting counts)
+            pd_table.drop(pd_table.columns[[10, 11, 12, 13]], axis=1, inplace=True)
+            # Rename headers
+            pd_table.columns = [
+                'Provincia', 'Stazione', Pollutant.PM10, Pollutant.PM2_5, Pollutant.NO2, Pollutant.O3, 'O3_8h',
+                Pollutant.BENZENE, Pollutant.CO, Pollutant.SO2
+            ]
+            # Split Station field into Station, Type
+            # TODO: Differentiate between Location and Name in Station field...
```

Comment:
On this, the keys are in italian but most likely I will switch also piemonte.py to english, also for the CSV column titles, if you can change it also here would be good! Thx!

Reply:
Ok, done.